godotproject/Objects/Rocket/Rocket.gd

```diff
@@ -59,17 +59,21 @@ func go_to_space():
 	var player = Global.get_player()
 	player.zoom_out(4)
 	player.trigger_screen_shake(10, 30, 8, 1)
-	$JetBoom.play()
+	$JetBoomNonLocational.play()
 	$Timer.start()
 	
 	
 func _on_Timer_timeout():
 	# Rocket has now left
 	if frog_on_board:
 		$VictorySoundPlayer.play()
+		# Prevent high score screen from going until the sound effect is done
+		Global.get_node("HighScoreScreenLayer/HighScoreScreen").wait_in_initial_state = true
```

Comment:
Can we add this to every highscore screen and not just at the end of the game? I know I was the one to implement this in the first place, I have it listed as a TODO under my polish all levels ticket, I just dont want to get merge conflicts if we both touch the highscore script

Reply:
This being the Victory sound? We could def do that

Reply:
yeah